Gems/GradientSignal/Code/Source/Editor/EditorImageGradientComponentMode.cpp

```diff
@@ -143,6 +144,33 @@ namespace GradientSignal
         ImageGradientModificationNotificationBus::Handler::BusDisconnect();
     }
 
+    void EditorImageGradientComponentMode::Reflect(AZ::ReflectContext* context)
+    {
+        AzToolsFramework::ComponentModeFramework::ReflectEditorBaseComponentModeDescendant<EditorImageGradientComponentMode>(context);
+    }
+
+    void EditorImageGradientComponentMode::RegisterActions()
+    {
+    }
```

Comment:
Suggestion: Maybe add a comment to this and BindActionsToMenus() to explain why they're empty but still exist.

Reply:
Done!